populateCouch.js

```diff
@@ -0,0 +1,54 @@
+require('dotenv').config();
+const nano = require('nano')(`http://admin:${process.env.NANOPZ}@localhost:5984`);
+
+let NANO_DB = 'test'
```

Comment:
Nothing wrong with these variables, but if you are already using process.env for the password, maybe worth it to add these there instead